pkgs/enchant/1.x.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, lib, fetchurl, aspell, pkgconfig, glib, hunspell, hspell }:
+
+stdenv.mkDerivation (finalAttrs: {
+  version = "1.6.1";
+  pname = "enchant";
+
+  src = fetchurl {
+    url = "https://github.com/AbiWord/enchant/releases/download/enchant-${finalAttrs.version}/enchant.tar.gz";
+    sha256 = "1xg3m7mniyqyff8qv46jbfwgchb6di6qxdjnd5sfir7jzv0dkw5y";
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [ aspell glib hunspell hspell ];
+
+  meta = {
+    description = "Generic spell checking library";
+    homepage = "https://abiword.github.io/enchant";
+    platforms = lib.platforms.unix;
+    license = lib.licenses.lgpl21;
```

Comment:
`lgpl21` is unclear, use `lgpl21Plus` or `lgpl21Only`. See https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206 for more details.

Reply:
Done, replaced with `lgpl21Only`.